packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -169,6 +169,11 @@ export function createAutocompleteDom<TItem extends BaseItem>({
       type: 'button',
       class: classNames.detachedCancelButton,
       textContent: translations.detachedCancelButtonText,
+      // Prevent environment `onTouchStart` behavior from closing the panel
+      // if it's initiated from this button
```

Comment:
Yes, we're stopping the touchStart event propagation to make sure the panel closing is initiated by the cancel button (via the `click` event) instead of the [environment/window `touchStart` event](https://github.com/algolia/autocomplete/blob/next/packages/autocomplete-core/src/getPropGetters.ts#L35-L73).

Reply:
Okay, so maybe the comment could be something like:

>Prevent `onTouchStart` from closing the panel
>since it should be initiated by `onClick` only

wdyt?